Hashlink.js

```diff
@@ -34,8 +35,65 @@ export class Cipher {
    *
    * @returns {Promise<string>} Resolves to a string that is a hashlink.
    */
-  async create({data, urls, transforms, meta}) {
-    throw new Error('Not implemented.');
+  async create({data, urls, transforms, meta = {}}) {
+    // ensure data or urls are provided
+    if(data === undefined && urls == undefined) {
+      throw new Error('Either `data` or `urls` must be provided.')
+    }
+
+    // transforms are provided
+    if(transforms === undefined) {
+      throw new Error('The hashlink creation `transforms` must be provided.')
+    }
+
+    if(urls) {
+      // ensure urls are an array
+      if(Array.isArray(urls) === false) {
+        urls = [urls];
+      }
+
+      // ensure all URLs are strings
+      urls.forEach(url => {
+        if(typeof url !== 'string') {
+          throw new Error('Each `url` must be a string. Value: ' + url, url);
+        }
+      });
+    }
+
+    // merge meta options with urls
+    meta = Object.assign(meta, {'url': urls});
```

Comment:
```suggestion
    meta = {...meta, url: urls};
```

Also, move this inside the `if(urls)` conditional above.